scripts/bundle_clusterfuzz.py

```diff
@@ -135,11 +135,17 @@
             tar.add(libbinaryen, arcname=f'lib/libbinaryen{suffix}')
 
             # The emsdk build also includes some more necessary files.
-            for name in [f'libc++{suffix}', f'libc++{suffix}.2', f'libc++{suffix}.2.0']:
-                path = os.path.join(binaryen_lib, name)
-                if os.path.exists(path):
-                    print(f'  ......... : {path}')
-                    tar.add(path, arcname=f'lib/{name}')
+            for lib in ['libc++', 'libmimalloc']:
+                for name in [
+                    f'{lib}{suffix}',
+                    f'{lib}{suffix}.2',
+                    f'{lib}{suffix}.2.0',
+                    f'{lib}{suffix}.2.2',
```

Comment:
Is there a better way than to hardcode these..?

Reply:
Maybe use [`glob`](https://docs.python.org/3/library/glob.html) to find all the paths matching `{lib}{suffix}*`?

Reply:
Yeah, I guess that would be more flexible. But in theory using `ldd` to find the actual dependencies of `wasm-opt` seems more robust... but I don't really understand the whole `.so, .so.2, .so.2.2` stuff with symlinks etc. `ldd wasm-opt` reports `libmimalloc.so.2`, but the other prefixes exist too.

Reply:
Yeah, I already have a PR out to update mimalloc to 2.2.3, so using ldd to be exact about it or a glob of some kind seems like what we want.
The single digit suffix (the soname) will change if we e.g. update to mimalloc v3 at some point. IIUC the idea behind the soname having just the major version is that you can expect binary compatibility between minor versions but not major versions. But since we aren't worrying about compatibility or independent updates or anything, I think a simple glob is easy and unlikely to cause problems.

 If we want this to work precisely on mac (I see you're also including dylibs here) we'd need to use `otool` instead of `ldd`. Although I don't currently have plans to switch to mimalloc on mac, because when I tried it on my laptop it wasn't any faster.

Reply:
Thanks, sounds like glob is best for now then - if we need different things on linux/mac then the complexity seems risky.